tests/e2e/specs/e2e.spec.ts

```diff
@@ -71,6 +71,14 @@ test.describe("pages", () => {
   });
 });
 
+test("it keeps the favicon between pages", async ({ page }) => {
+  await page.goto("./vite-page");
```

Comment:
i think it makes more sense to use vite-page and head-test as our sample pages and leave the react body injection page for its sole purpose